connectors/src/admin/cli.ts

```diff
@@ -383,6 +391,61 @@ const slack = async (command: string, args: parseArgs.ParsedArgs) => {
 
       break;
     }
+
+    case "clean-nango-connection-id": {
+      const slackConfigurations = await SlackConfiguration.findAll();
+      const connections = await nango_client().listConnections();
+      const slackConnections = connections.connections.filter(
+        (connection: {
+          id: number;
+          connection_id: string;
+          provider: string;
+          created: string;
+        }) => connection.provider === NANGO_SLACK_CONNECTOR_ID
+      );
+
+      let connectionDetail,
+        slackTeamId: string | null = null;
+
+      const askQuestion = async (query: string): Promise<string> => {
+        const rl = readline.createInterface({
+          input: process.stdin,
+          output: process.stdout,
+        });
+
+        return new Promise((resolve) => {
+          rl.question(query, (answer) => {
+            rl.close();
+            resolve(answer);
+          });
+        });
+      };
+
+      for (const connection of slackConnections) {
+        connectionDetail = await nango_client().getConnection(
+          connection.provider,
+          connection.connection_id
+        );
+
+        slackTeamId = connectionDetail.credentials.raw.team.id;
+
+        if (!slackConfigurations.find((sc) => sc.slackTeamId === slackTeamId)) {
+          const answer: string = await askQuestion(
+            `Do you want to delete connection ${connection.connection_id} for team ${slackTeamId}? (y/N) `
+          );
+          if (answer.toLowerCase() !== "y") {
+            continue;
+          }
+
+          console.log("Deleting connection...");
+          await nangoDeleteConnection(
```

Comment:
This won't revoke the token will it? We probably want to revoke the token as well. We can have it for slack only and error otherwise (and not delete the nango connection if that's the case for other platforms)

Reply:
To revoke a Slack token I think we need to call this api endpont:
https://api.slack.com/methods/auth.revoke

We should not uninstall because if the user logged in with another token, it will uninstall our bot from their workspace (as well as revoke the token).

Reply:
I called our new method `uninstallSlack` instead